### ISSUE TITLE
Deploy MS with the last release

### ISSUE THREAD

**Issue by chpicone** [geosolutions-it/MapStore2-C098#77]
Deploy the MS to the last release (with the MapTip (hover) functionality)

**Comment by MV88** [comment]
[here](http://demo.geo-solutions.it/share/RegioneToscana/) there is the war with MapStore updated to 2020.02.xx

**Comment by randomorder** [comment]
@giacomoc92 
The goad is to update the MapStore instance "In Test"
If I remember correctly they don't have an actual est env or a test VM for MapStore (please correct me if I am wrong) so the idea is to "clone" the existing prod instance and use this clone to test the updgrade.

@tdipisa @MV88 does this Mapstore update require a geostore database schema upgrade?

**Comment by randomorder** [comment]
( Duplicate of https://github.com/geosolutions-it/DevOps/issues/234 ?  @chpicone )

**Comment by chpicone** [comment]
@giacomoc92 if I remember correctly you already prepared a parallel tomcat instance where we can deploy the new version and make some test, right?

**Comment by giacomoc92** [comment]
This is servizio idrologico right? last time i worked on it they had a test environment (TIX) which is a clone of the VMs in the production environment (SIR)

**Comment by chpicone** [comment]
@giacomoc92 Yes it is for Servizio Idrologico.
Las time you said me that you already installed an addictional tomcat on their test environment.

**Comment by chpicone** [comment]
@giqacomc92: Yes it is for Servizio Idrologico. You can proceed updating the test environment.
Alessandro Santucci wrote me to use server with IP: 10.159.212.210 (I think it is TIX)

**Comment by randomorder** [comment]
I think that is a Clone of Either TIX or SIR
@giacomoc92 move on with the update on the machine with IP `10.159.212.210` then update the Deployment diagram to include this new VM.

**Comment by giacomoc92** [comment]
Update is completed.

To reach the instance: Get the [openvpn configuration](https://docs.google.com/document/d/16mtwdgvsjlVA-khTdboeA2MYqYiJ61RXeIovj8dZ0GY/edit) for their environment and use it ([instructions](https://docs.google.com/document/d/1AGXq9PsjtfQKetPgZRW1VPqhdmB_6vCdRup3AqOLiHI/edit#heading=h.oyeixruyy9n2)).

The mapstore instance should then be reachable at http://10.159.212.210/mapstore

The machine is a clone of TIX, so the info (credentials, etc...) in the document, should apply to it too.

**Comment by chpicone** [comment]
@giacomoc92 thx.

**Comment by randomorder** [comment]
To clarify: "Update is completed." in the TEST environment

**Comment by chpicone** [comment]
Yes correct. We need to proceed with end 2 end tests #79, then inform the customer to make their internal tests and finally to deploy in prod environment

**Comment by randomorder** [comment]
waiting for tests to be finished before moving on with PROD

**Comment by chpicone** [comment]
@randomorder we can proceed with a new installation of the war in the test environment.

**Comment by randomorder** [comment]
@giacomoc92 Please go ahead with the deployment of [the updated MS WAR](http://demo.geo-solutions.it/share/RegioneToscana/MapStore2-C098_2020-07-20.war) in the TEST env

**Comment by giacomoc92** [comment]
Deployment in test is done

**Comment by chpicone** [comment]
@giacomoc92 thx.
Even tests are passed ([here](https://github.com/geosolutions-it/MapStore2-C098/issues/79)) 
@randomorder  we are waiting the customer (I wrote an email to Alessandro Santucci) to schedule the deploy in the production environment

**Comment by randomorder** [comment]
news @chpicone ?

**Comment by chpicone** [comment]
@randomorder we agreed with the customer  for an improvement of the application (with a new MapTip function).
So I will open another issue to install this new version of MS.
